### ISSUE TITLE
IPN not VERIFIED

### ISSUE THREAD

**Issue by mkasson** [srmklive/laravel-paypal#80]
Thanks for the great package srmklive!  I've got an express checkout flow running in the sandbox and all seems to be working except IPN verification.

I do `setExpressCheckout` before paypal and `DoExpressCheckoutPayment` when the flow comes back to my code.  My PDT POSTed to payment/success shows status of Completed and I can verify the transactions in my Buyer sandbox account.  I am using the code for IPN verification exactly as written in the README but I do not get `$response === 'VERIFIED'`.

I have emailed to myself `print_r($response, true)` and I get a guzzle stream:
`GuzzleHttp\Psr7\Stream Object ( [stream:GuzzleHttp\Psr7\Stream:private] => Resource id #379 [size:GuzzleHttp\Psr7\Stream:private] => [seekable:GuzzleHttp\Psr7\Stream:private] => 1 [readable:GuzzleHttp\Psr7\Stream:private] => 1 [writable:GuzzleHttp\Psr7\Stream:private] => 1 [uri:GuzzleHttp\Psr7\Stream:private] => php://temp [customMetadata:GuzzleHttp\Psr7\Stream:private] => Array ( ) )`

Not sure how to take it further. Any guidance would be greatly appreciated.

**Comment by srmklive** [comment]
I would say your API Credentials are incorrect. I just tested IPN on the [demo app](https://laravel-paypal-demo.srmk.info/) and i am getting `VERIFIED` response. 

It might be an issue with your sandbox credentials.

**Comment by mkasson** [comment]
Thanks for the fast response!  They're copied straight off the developer.paypal.com Dashboard.  Under Accounts, I selected my Business account, then Profile.  From under API Credentials I copied the username, password and signature.  I'm assuming the signature is what your docs call the "secret".
I also verfied them on sandbox.paypal.com in my profile for my sandbox business account (under My Selling Tools, API Access, then NVP/SOAP Access where it's also username, password and signature).

Wouldn't the express checkout calls (before and after paypal) have failed if the creds were bad?

Could there be something missing from $data? I only give what appears to be the minimum needed.
